recipes/keyrings.alt/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "keyrings.alt" %}
```

Comment:
I think it was decided that the docs are in error.  The part about periods not being allowed was added pretty recently.